modules/rostests/winetests/user32/CMakeLists.txt

```diff
@@ -48,6 +48,6 @@ endif()
 
 add_pch(user32_winetest precomp.h "${PCH_SKIP_SOURCE}")
 # some tests need to be run from an app compiled as GUI
```

Comment:
If this is no longer true, you should remove this comment.
That said, has anything changed to make this unnecessary now?

Reply:
TBH, I don't know and this comment / the commit message don't help. I don't see any regression on Test WHS though, so I think it should be ok.

Reply:
Current PR reverts 559bf9639468e83ff34a6f06d3501ccb1efdccb3.

Reply:
@zefklop could shed light on that, then...